.github/workflows/pythonapp.yml

```diff
@@ -0,0 +1,45 @@
+name: Run pytest
+
+on:
+  push:
+    branches-ignore: 
+      - gh-pages
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
```

Comment:
The default `pythonpackage.yml` action uses a matrix build to test on 2.7, 3.5, 3.6, 3.7. I like that, and I'm wondering if there's a good way to keep that. I realize the reports might end up needing to be merged somehow - maybe there's a way to dump all the reports into the same place and push with a post-action?

It's important to me that this package is tested with Python 2.7; although 2.7 is officially EOL tomorrow, there's still a nontrivial amount of software targeting 2.7, and keeping compatibility is not that difficult yet. I will revisit this if/when supporting 2.7 becomes cumbersome or there's another compelling reason to go 3.x-only.

Reply:
Oh! I was using pythonapp.yml, not pythonpackage.yml. I'll change that.

Reply:
Testing in Python 3.5 has a high failure rate.
```
=================== 10 failed, 3 passed, 1 skipped in 12.17s ===================
```